powershell/resources/assets/check-dependencies.ps1

```diff
@@ -41,8 +41,8 @@ $resourceModule = Join-Path $HOME '.PSSharedModules\Resources\Az.Resources.TestS
 if ($Resources.IsPresent -and (-not (Test-Path -Path $resourceModule))) {
   Write-Host -ForegroundColor Green "Building local Resource module used for test..."
   Set-Location $resourceDir
-  autorest-beta .\readme.md --output-folder=$HOME/.PSSharedModules/Resources
+  $null = autorest-beta .\readme.md --output-folder=$HOME/.PSSharedModules/Resources
```

Comment:
Is $null used to avoid too many output info? why?

Reply:
Yes, it is. Since the customized version of Resources module is quite stable, there should be no errors happen for these steps. So try to suppress the output here.